src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -2734,6 +2826,15 @@ namespace Microsoft.FSharp.Core
         /// <param name="y">The second parameter.</param>
         ///
         /// <returns>The result of the comparison.</returns>
+        /// 
+        /// <example id="compare-less-than-example">
+        /// <code lang="fsharp">
+        ///  1 < 5               // Evaluates to true
+        ///  5 < 5               // Evaluates to false
+        ///  (1, "a") < (1, "z") // Evaluates to true
```

Comment:
`&lt;` needed see https://stackoverflow.com/questions/33418668/how-to-properly-escape-the-content-of-a-code-example-in-xml-documentation